### DISCUSSION TITLE
Adguard DNS의 List-KR 필터와 Adguard의 필터는 뭐가 다른가요?

### DISCUSSION THREAD

**Discussion by gminshin** [List-KR/community#63] [Q&A]
최근에 Adguard VPN을 구매해서 Adguard Personal DNS가 오픈되었는데,

해당 DNS를 관리하는 사이트에 들어가보니 필터로 List-KR을 추가할 수 있어 문의드립니다.

둘이 뭐가 다른가요?

**Comment by piquark6046** [comment]
AdGuard DNS에 있는 List-KR 필터와 AdGuard 필터를 포함한 HostList는 전체적으로 차단해도 별 문제가 없거나 주 목적인 광고/추적 같은 그러한 목적인 도메인만 차단합니다.

AdGuard DNS에 있는 List-KR 필터는 원본 List-KR 필터를 Hostlist화 한 것입니다.
AdGuard DNS에 있는 AdGuard 필터는 https://github.com/AdguardTeam/AdGuardSDNSFilter 를 참고하시면 됩니다.

**Comment by gminshin** [comment]
그렇군요. 그럼 그 두 필터를 동시에 사용해도 되나요??

**Reply by piquark6046** [reply]
네.
